specs/protocol/configurability.md

```diff
@@ -0,0 +1,79 @@
+| [Resource config](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L338-L340)                       | Config for the EIP-1559 based curve used for the deposit gas market.                                                         | [System Config Owner](#admin-roles)                 |
+| Sequencing window                     | Maximum allowed batch submission gap, after which L1 fallback is triggered in derivation.                                    |                                     |
+| [Start block](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L184)                           | Block at which the system config was initialized the first time.                                                             | [L1 Proxy Admin](#admin-roles)                      |
+| Superchain target                     | Choice of cross-L2 configuration. May be omitted in isolated OP Stack deployments. Includes SuperchainConfig and ProtocolVersions contract addresses. |            |
+
+## Policy Parameters
+
+| Config Property                       | Description                                                                                                                  | Administrator                       |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| Batch submission frequency            | Frequency with which batches are submitted to L1 (see [Batcher Transaction](../glossary.md#batcher-transaction)).            |                                     |
+| Compression ratio                     | How much compression the batch submitter applies to batches before submission (see [Channel](../glossary.md#channel)).       |                                     |
+| Implementation Fee vault contracts    | Implementation contracts that sit behind proxies and dictate how user fees are distributed.                                  |                                     |
+| [Output frequency](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/L2OutputOracle.sol#L104)                      | Frequency with which output roots are submitted to L1.                                                                       | [L1 Proxy Admin](#admin-roles)                      |
+
+## Admin Roles
+
+| Config Property                       | Description                                                                                                                  | Administrator                       | Administers                         |
```

Comment:
Noting that my requested changes above would need to be reflected in removals here. It would be really nice to have a script to populate the `administers` field, but nice to have

Reply:
Sounds really useful, do we have an existing pattern for using scripts with `.md` files in the specs repo?